netlify.toml

```diff
@@ -20,14 +20,13 @@
     Referrer-Policy = "origin-when-cross-origin"
     Strict-Transport-Security = "max-age=2592000"
     Permissions-Policy = "vibrate=(), geolocation=(), midi=(), notifications=(), push=(), microphone=(), camera=(), magnetometer=(), gyroscope=(), speaker=()"
-    Basic-Auth = "adapt:letmein2021"
```

Comment:
We will still want to have the basic auth here as dev is currently set to generate unpublished content.